api-2.2/src/main/java/org/openmrs/module/coreapps/htmlformentry/EncounterDiagnosesElement.java

```diff
@@ -0,0 +1,361 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
```

Comment:
This is an older version of the license.

Reply:
@dkayiwa i am on it.

Reply:
@dkayiwaI have updated the license.